packages/clis/generator/graph/check-graph.ts

```diff
@@ -25,12 +28,16 @@ interface Unrouteable {
 export async function checkGraph(
   graph: Map<string, Neighbors>,
   tsMapData: MappedData,
+  map: 'usa' | 'europe',
 ) {
-  const { nodes, companies, prefabs } = tsMapData;
+  const { nodes, companies, prefabs, cities } = tsMapData;
 
-  // check that all companies can be reached from some random company.
-  const allCompanies = [...companies.values()].filter(company =>
-    prefabs.has(company.prefabUid.toString(16)),
+  // check that all companies in known cities can be reached from some
+  // random company.
+  const allCompanies = [...companies.values()].filter(
+    company =>
+      cities.has(company.cityToken) &&
```

Comment:
This extra filtering condition cleans up a lot of `generator graph` log spam about unreachable, future-DLC cities.